packages/core/helpers/ver/4.24.10.ts

```diff
@@ -11,9 +12,13 @@ export const removeMultiSelectPropFromMultiselect = newConfig => {
   }
 }
 
+export const setXAxisLabelOffsetToZero = newConfig =>
+  editConfigKeys(newConfig, [{ path: ['xAxis', 'labelOffset'], value: 0 }])
+
 const update_4_24_10 = config => {
   const ver = '4.24.10'
-  const newConfig = _.cloneDeep(config)
+  let newConfig = _.cloneDeep(config)
+  newConfig = setXAxisLabelOffsetToZero(config)

```

Comment:
I was hoping I could start using a pattern of not mutating objects when it makes sense. I like that as a best practice of functional programming because it makes bugs easier for me to trace and the code easier to follow imo. I have a cloneDeep in the editConfigKeys function, so it's baked somewhat into it but I can change it if you object (pun slightly intended). Also game to discuss how we all feel about immutability as a group and rewrite this for now so It matches the existing pattern.

Reply:
Yeah, there is no real risk here, just a personal preference. I'll mutate for now and hold out for a larger refactor.